### ISSUE TITLE
Home page. For using a screen reader, Create an Account is a button on the Home page, but Sign In is a link. They should both either be buttons or links

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#1639]
## Description of issue

For using a screen reader, Create an Account is a button on the Home page, but Sign In is a link. They should both either be buttons or links

Describe what is failing
- include what is failing
- include an image
<img width="1215" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNjFwQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1d0adf8fdaf0bdd8c14980ab16d947ba78f0d30a/Screenshot%202024-08-19%20at%209.48.39%E2%80%AFAM.png" alt="Screenshot 2024-08-19 at 9.48.39 AM.png" />

- include context
When doing usability testing for Terms of Use interventions on Account Creation and Sign In, a user navigating with a screen reader had some difficulty finding the sign in because it was a link and not a button

## Potential fix

- What change could fix this issue?
Use both links or both buttons for predicability and consistency
- What nuggets of information could help us fix the issue
Use WCAG 2.1/2.2 AA criteria

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

## Acceptance Criteria 
- [x] We are consistent in our usage of links and buttons so screenreaders have predictability of how our app will behave
- [x] We follow best practices and guidance available to date
- [x] Change is made, and QA'd in Staging then Prod for both En and FR

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1639?planning-poker) with Zenhub @amazingphilippe @andrewleith

**Comment by amazingphilippe** [comment]
Removed a script that changed how links behaved based on styling. We now let links be links, even if they have a background. 

This follows best-practices and separates form and function. Users may continue to think links are buttons, but we won't be introducing dubious html markup that might conflict with some assistive technology
